local-cli/link/unlink.js

```diff
@@ -94,10 +107,12 @@ const unlinkDependencyIOS = (iOSProject, dependency, packageName, iOSDependencie
 function unlink(args, config) {
   const packageName = args[0];
 
-  var project;
-  var dependency;
+  let platforms;
+  let project;
+  let dependency;
 
   try {
+    platforms = config.getPlatformConfig();
```

Comment:
@grabbou - here is one example of where we need to call `getPlatformConfig()` on the `RNConfig`.